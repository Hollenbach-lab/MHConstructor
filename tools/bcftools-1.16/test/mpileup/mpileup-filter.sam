@HD	VN:1.4	GO:none	SO:coordinate
@SQ	SN:1	LN:150	M5:de3677cffe7c371fbb69fd44ce9bc181	UR:/nfs/users/nfs_p/pd3/git/stools/bcftools/test/mpileup/mpileup-SCR.fa
@RG	ID:rg	SM:sample
HS40_17920:7:2214:9967:66664#16	99	1	64	60	75M	*	0	167	CCTAGGTCCCCCCGTGCCACCATGATGCCGTCGCTCACCTCCAGGATTTCATCAAACCTGAGAGGTTGGGAGAAT	@A??DE?FEEEEEA@HGEBEDCBHBBHGEA@FAGDFCFEDFECGFBBBBFCBFCCCFEDHBGBGF@BHFEAC>??	BC:Z:TCCGTCTT	MC:Z:75M	BD:Z:LLMMQPOPNJJJJLMKNMMKMMNNNMNNMLMNMMNKMKNLLMNONMNKEMNOONMDMNMOOMMMOONNPLOMMMM	PG:Z:MarkDuplicates	BI:Z:PPQQRPRRPLLLLOQORPQNQQQRRQRRPOQQPPROQOQPOOQRPQRPIPRRRRPHQRQSSPPQQSQSSOSPPPQ	MQ:i:60	AS:i:75	XS:i:24	QT:Z:BBBBBFFF	mc:i:155264286	ms:i:2755	BQ:Z:@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@	MD:Z:75	NM:i:0	RG:Z:rg
HS40_17920:7:2214:9967:66664#16	3	1	64	0	75M	*	0	167	CCTAGGTCCCCCCGTGCCACCATGATGCCGTCGCTCACCTCCAGGATTTCATCAAACCTGAGAGGTTGGGAGAAT	@A??DE?FEEEEEA@HGEBEDCBHBBHGEA@FAGDFCFEDFECGFBBBBFCBFCCCFEDHBGBGF@BHFEAC>??	BC:Z:TCCGTCTT	MC:Z:75M	BD:Z:LLMMQPOPNJJJJLMKNMMKMMNNNMNNMLMNMMNKMKNLLMNONMNKEMNOONMDMNMOOMMMOONNPLOMMMM	PG:Z:MarkDuplicates	BI:Z:PPQQRPRRPLLLLOQORPQNQQQRRQRRPOQQPPROQOQPOOQRPQRPIPRRRRPHQRQSSPPQQSQSSOSPPPQ	MQ:i:60	AS:i:75	XS:i:24	QT:Z:BBBBBFFF	mc:i:155264286	ms:i:2755	BQ:Z:@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@	MD:Z:75	NM:i:0	RG:Z:rg
