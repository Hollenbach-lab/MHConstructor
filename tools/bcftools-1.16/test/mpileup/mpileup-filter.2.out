##fileformat=VCFv4.2
##FILTER=<ID=PASS,Description="All filters passed">
##contig=<ID=1,length=150>
##ALT=<ID=*,Description="Represents allele(s) other than observed.">
##INFO=<ID=INDEL,Number=0,Type=Flag,Description="Indicates that the variant is an INDEL.">
##INFO=<ID=IDV,Number=1,Type=Integer,Description="Maximum number of raw reads supporting an indel">
##INFO=<ID=IMF,Number=1,Type=Float,Description="Maximum fraction of raw reads supporting an indel">
##INFO=<ID=DP,Number=1,Type=Integer,Description="Raw read depth">
##INFO=<ID=VDB,Number=1,Type=Float,Description="Variant Distance Bias for filtering splice-site artefacts in RNA-seq data (bigger is better)",Version="3">
##INFO=<ID=RPBZ,Number=1,Type=Float,Description="Mann-Whitney U-z test of Read Position Bias (closer to 0 is better)">
##INFO=<ID=MQBZ,Number=1,Type=Float,Description="Mann-Whitney U-z test of Mapping Quality Bias (closer to 0 is better)">
##INFO=<ID=BQBZ,Number=1,Type=Float,Description="Mann-Whitney U-z test of Base Quality Bias (closer to 0 is better)">
##INFO=<ID=MQSBZ,Number=1,Type=Float,Description="Mann-Whitney U-z test of Mapping Quality vs Strand Bias (closer to 0 is better)">
##INFO=<ID=NMBZ,Number=1,Type=Float,Description="Mann-Whitney U-z test of Number of Mismatches within supporting reads (closer to 0 is better)">
##INFO=<ID=SCBZ,Number=1,Type=Float,Description="Mann-Whitney U-z test of Soft-Clip Length Bias (closer to 0 is better)">
##INFO=<ID=FS,Number=1,Type=Float,Description="Phred-scaled p-value using Fisher's exact test to detect strand bias">
##INFO=<ID=SGB,Number=1,Type=Float,Description="Segregation based metric.">
##INFO=<ID=MQ0F,Number=1,Type=Float,Description="Fraction of MQ0 reads (smaller is better)">
##INFO=<ID=I16,Number=16,Type=Float,Description="Auxiliary tag used for calling, see description of bcf_callret1_t in bam2bcf.h">
##INFO=<ID=QS,Number=R,Type=Float,Description="Auxiliary tag used for calling">
##FORMAT=<ID=PL,Number=G,Type=Integer,Description="List of Phred-scaled genotype likelihoods">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	sample
1	100	.	A	<*>	0	.	DP=1;I16=1,0,0,0,34,1156,0,0,60,3600,0,0,25,625,0,0;QS=1,0;MQ0F=0	PL	0,3,34
